go/cmd/vttablet/cli/cli.go

```diff
@@ -143,7 +143,6 @@ func run(cmd *cobra.Command, args []string) error {
 
 	qsc, err := createTabletServer(ctx, env, config, ts, tabletAlias, srvTopoCounts)
 	if err != nil {
-		ts.Close()
```

Comment:
I think we just need to merge in origin/main rather than doing this here.

Reply:
Done